api/src/pcapi/core/offerers/api.py

```diff
@@ -277,11 +278,19 @@ def link_venue_to_reimbursement_point(
             timestamp,
         )
         db.session.add(current_link)
+        logger.info(
+            "VenueReimbursementPointLink has ended",
+            extra={"venue_id:": venue.id, "former_reimbusementPointId": current_link.reimbursementPointId},
```

Comment:
(détail / style + typo) Est-ce un chameau ? Est-ce un serpent ? Est-ce un chapent ou un sermeau ?! Ou bien un semeau puisqu'il manque un "r" ? :)

```suggestion
            extra={"venue_id:": venue.id, "former_reimbursement_point": current_link.reimbursementPointId},
```

(idem plus bas dans l'autre log ajouté)


![chapent](https://user-images.githubusercontent.com/471321/218747016-cbb968e3-3720-408a-be9c-ac285ce16c0b.jpg)

Reply:
Merci d'avoir remarqué cette tyhpon.